mypy/test/testconfigparser.py

```diff
@@ -0,0 +1,168 @@
+import os
+import tempfile
+from unittest import TestCase, main
+
+from mypy.config_parser import parse_config_file
+from mypy.options import Options
+
+
+class TestConfigParser(TestCase):
+    def test_parse_config_file_with_single_file(self) -> None:
+        """A single file should be correctly parsed."""
+        with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
You shouldn't normally manage temporary dirs manually in pytest. It provides a built-in `tmp_path` fixture. Use it instead.